firmware/screen/src/main.cpp

```
@@ -195,6 +264,109 @@ static void refresh_ui() {
+static lv_disp_draw_buf_t lv_draw_buf;
+static lv_color_t*        lv_buf_a = nullptr;
+static lv_color_t*        lv_buf_b = nullptr;
+static lv_disp_drv_t      lv_disp_drv;
+static lv_indev_drv_t     lv_indev_drv;
+static uint32_t           lv_last_tick_ms = 0;
+static uint32_t           lv_next_canvas_flush_ms = 0;
+static bool               lv_canvas_dirty = false;
+
+static void lv_flush_cb(lv_disp_drv_t* drv, const lv_area_t* area, lv_color_t* color_p) {
+  if (gfx) {
+    uint32_t w = (area->x2 - area->x1 + 1);
```

Clang (14.0.6)

[warning] variable 'w' is not initialized

(cppcoreguidelines-init-variables)